src/Aspire.Hosting.Azure.Storage/AzureQueueStorageQueueResource.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Azure;
+using Azure.Provisioning;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// A resource that represents an Azure Blob Storage queue.
+/// </summary>
+/// <param name="name">The name of the resource.</param>
+/// <param name="queueName">The name of the queue.</param>
+/// <param name="parent">The <see cref="AzureQueueStorageResource"/> that the resource is stored in.</param>
+public class AzureQueueStorageQueueResource(string name, string queueName, AzureQueueStorageResource parent) : Resource(name),
```

Comment:
This name... is a mouth full of "Queue".

Reply:
Yes, we have a "naming" issue, I raised this point in [Teams](https://teams.microsoft.com/l/message/19:uuDJgMCR7JeFSM-0RK2xRW5mAq2WFNfUK0TLmYDvAU01@thread.tacv2/1746584871241?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=f2e0b0b6-f130-46fa-8ec6-2ef4a7168ed2&parentMessageId=1746584871241&createdTime=1746584871241&ngc=true).